src/main/java/caculator/Calculator.java

```diff
@@ -0,0 +1,90 @@
+package caculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class Calculator {
+
+    // "문자열을 분리"
+    int splitString(String text) throws Exception {
```

Comment:
```suggestion
    // "문자열을 분리"
    public int splitString(String text) throws Exception {
```
이 부분도 앞서 말씀드렸던 내용입니다. 외부에서 접근가능한 메소드는 public을 붙여주는 것이 좋습니다.